000-vip-init.php

```diff
@@ -16,8 +16,12 @@
  * good reason not to.
  */
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit();
+}
+
 // Execute the healthcheck as quickly as possible
-if ( '/cache-healthcheck?' === $_SERVER['REQUEST_URI'] ) {
+if ( isset( $_SERVER['REQUEST_URI'] ) && '/cache-healthcheck?' === $_SERVER['REQUEST_URI'] ) {
```

Comment:
> Detected usage of a possibly undefined superglobal array index: $_SERVER['REQUEST_URI']. Use isset() or empty() to check the index exists before using it

This could trigger a Notice if the plugin is invoked from a CLI script.